src/client/js/main.js

```diff
@@ -1,5 +1,23 @@
-// 아래는 메인 페이지 관련 js만드셨 을때 예제입니다. 주석을 지우고 하시면 됩니다.
-
 import "../scss/styles.scss";
+import Category from "./category.js";
+import FloatingButton from "./floatingButton.js";
+
+const categoryComponent = new Category(document.querySelector(".categoryList"));
+
+if(document.querySelector(".main-wrap section").scrollHeight > window.innerHeight - 359) {
+  const floatingButton = new FloatingButton().getButton();
+  floatingButton.className = "main_floatingButton";
+  document.querySelector("body").appendChild(floatingButton);
 
+  floatingButton.addEventListener('click', (e) => {
```

Comment:
저희 다른 팀원이 event로 명시해주셨으니까 약어 대신 event로 풀 표기해주시면 통일성이 맞아 보입니다. 
```suggestion
  floatingButton.addEventListener('click', (event) => {
```

Reply:
통일성 맞추려면 event로 사용하는게 맞겠네요!
관련 코드 수정하겠습니다~~